extensions/amp-sticky-ad/0.1/amp-sticky-ad.js

```diff
@@ -125,6 +140,37 @@ class AmpStickyAd extends AMP.BaseElement {
       });
     }
   }
+
+  /**
+   * The function that add a close button to sticky ad
+   * @private
+   */
+  addCloseButton_() {
+    if (this.closeButton_ && this.boundOnClick_) {
+      return;
+    }
+    const closeButton = this.getWin().document.createElement('div');
+    closeButton.setAttribute('class', '-amp-sticky-ad-close-button');
+    closeButton./*OK*/innerHTML = '&#10006';
+    this.element.appendChild(closeButton);
+    this.closeButton_ = closeButton;
+    if (!this.boundOnClick_) {
+      this.boundOnClick_ = this.onClick_.bind(this);
+    }
+    closeButton.addEventListener('click', this.boundOnClick_);
+  }
+
+  /**
+   * The listener function that listen to click event and dismiss sticky ad
+   * @private
+   */
+  onClick_() {
```

Comment:
Let's rename to make it clear what's actually being clicked. Reading it on the top-level sounds like it's a click on the `amp-sticky-ad`.